patches/nginx-1.11.2-balancer_status_code.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/src/http/ngx_http_upstream.c b/src/http/ngx_http_upstream.c
+index 0f6b3ae..56d44fc 100644
+--- a/src/http/ngx_http_upstream.c
++++ b/src/http/ngx_http_upstream.c
+@@ -1368,6 +1368,11 @@ ngx_http_upstream_connect(ngx_http_request_t *r, ngx_http_upstream_t *u)
+         return;
+     }
+ 
```

Comment:
Need to introduce a special `HAVE_ARBITRARY_BALANCER_STATUS_CODE_PATCH` macro in this patch.